torchtune/models/flux/_model_builders.py

```diff
@@ -67,7 +69,154 @@ def flux_1_autoencoder(
     )
 
     return FluxAutoencoder(
-        img_shape=(ch_in, resolution, resolution),
         encoder=encoder,
         decoder=decoder,
     )
+
+
+def flux_1_dev_flow_model():
+    """
+    Flow-matching model for FLUX.1-dev
+
+    Returns:
+        FluxFlowModel
+    """
+    return FluxFlowModel(
+        in_channels=64,
+        out_channels=64,
+        vec_in_dim=768,
+        context_in_dim=4096,
+        hidden_size=3072,
+        mlp_ratio=4.0,
+        num_heads=24,
+        depth=19,
+        depth_single_blocks=38,
+        axes_dim=[16, 56, 56],
+        theta=10_000,
+        qkv_bias=True,
+        use_guidance=True,
+    )
+
+
+def flux_1_schnell_flow_model():
+    """
+    Flow-matching model for FLUX.1-schnell
+
+    Returns:
+        FluxFlowModel
+    """
+    return FluxFlowModel(
+        in_channels=64,
+        out_channels=64,
+        vec_in_dim=768,
+        context_in_dim=4096,
+        hidden_size=3072,
+        mlp_ratio=4.0,
+        num_heads=24,
+        depth=19,
+        depth_single_blocks=38,
+        axes_dim=[16, 56, 56],
+        theta=10_000,
+        qkv_bias=True,
+        use_guidance=False,
```

Comment:
Noob q: the only diff between dev and schnell is that dev uses guidance and schnell doesn't?